airbyte-config/models/src/main/java/io/airbyte/config/EnvConfigs.java

```diff
@@ -177,6 +178,26 @@ public String getTemporalHost() {
         .collect(Collectors.toSet());
   }
 
+  @Override
+  public String getS3LogBucket() {
```

Comment:
would prefer a `LoggingConfig` object or something to make it clear that these are grouped together. Also maybe `LOG_AWS_ACCESS_KEY_ID`, etc.

Not blocking this since you're doing more refactoring after.

Reply:
oh good idea. will throw this in the follow up PR